torch/utils/data/dataset.py

```diff
@@ -299,6 +311,9 @@ def __getitem__(self, idx):
     def __len__(self):
         return len(self.indices)
 
+    def _getitems(self, idx):
```

Comment:
Why not just add `__getitems__` and raise Error if `self.dataset` doesn't have `__getitems__` implemented?

Reply:
Then, we can do the similar thing for `Subset.__getitems__` by calling `__getitems__` if it's available. Otherwise, calling `self.dataset[idx]` by iterating `idx` from `possibly_batched_index`

Reply:
good idea